src/main/java/org/folio/rest/service/ProcessorHelper.java

```diff
@@ -0,0 +1,12 @@
+package org.folio.rest.service;
+
+import io.vertx.core.json.JsonObject;
+
+public class ProcessorHelper {
+
+  private ProcessorHelper() {}
+
+  public static String[] getFunctionsFromCondition(JsonObject condition) {
+    return condition.getString("type").split(",");
+  }
```

Comment:
If you trim() each String here you don't need to do it multiple times afterwards.

Reply:
I just moved existing code to another place. I see your point, but the easy solution is not available in the String class. I would have to iterate over the String[] to trim() afterwards. It's not the moment to decide whether such is what I actually want. I don't want to alter functionality in this stage, unless all the consequences are absolutely clear.